hana/common.sls

```diff
@@ -20,13 +20,6 @@ install_required_packages:
 
 {% endif %}
 
-# Temporary solution to import the needed packages
-add-saphana-repo:
-  pkgrepo.managed:
-    - name: saphana
-    - baseurl: https://download.opensuse.org/repositories/home:xarbulu:sap-deployment/SLE_12_SP4/
```

Comment:
@arbulu89 This is still needed, but the should be changed to the new repo: https://download.opensuse.org/repositories/network:/ha-clustering:/Factory/SLE_12_SP4/

Reply:
@diegoakechi As discussed, the repositories must be added before using the formula as prerequisite

Reply:
OK. It makes the formula clean.